cypress/integration/todo_mvc/clear_all.js

```diff
@@ -0,0 +1,90 @@
+/// <reference types="cypress" />
+import faker from "faker";
+import selectors from "./selectors";
+
+describe("clear all the todos from the list", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.addToDos(5);
+  });
+
+  it("checks there is no clear all option when no todos are marked as completed", () => {
+    cy.log("open the tab COMPLETED")
+      .get(selectors.filter_options)
+      .find(selectors.tab_completed)
+	  .click()
+	  
+      .log("assert no items are marked as completed")
```

Comment:
The log is not quite accurate for what the next assertion really is. The assertion checks that there are no todos at all, regardless of their status.